parsely-javascript.php

```diff
@@ -1,6 +1,21 @@
 <?php if(!isset($parselyOptions['apikey']) || empty($parselyOptions['apikey'])) { return; } ?>
 
 <!-- START Parse.ly Include: Standard -->
+<script>
+    PARSELY = {
+        onload: function() {
+            var rootUrl = 'https://api.parsely.com/v2/profile?apikey=<?php echo esc_html($parselyOptions["apikey"]); ?>';
+            var uuid = '&uuid=' + PARSELY.config.parsely_site_uuid;
+            var requestUrl = rootUrl + uuid + '&url=' + window.location.href;
+            console.log(requestUrl);
```

Comment:
stray `console.log`?

Reply:
right you are, one day I will successfully submit a PR without missing a logging statement somewhere